### ISSUE TITLE
vapp property feature in vsphere_virtual_machine resource not working in v1.2.0 release

### ISSUE THREAD

**Issue by charandas** [terraform-providers/terraform-provider-vsphere#374]
Hi there,

This is regarding the vapp feature released in 1.2 in #303. It does not work when using the official binaries, whether I download them [manually](https://releases.hashicorp.com/terraform-provider-vsphere/1.2.0/), or allow `terraform init` to set it up for me.

It however **does work**, when I build the current `master` locally and provide the path through `$HOME/.terraformrc`.

### Terraform Version

``` ❯ terraform version
Terraform v0.11.2
+ provider.ct (unversioned)
+ provider.rancher v1.2.0
+ provider.template v1.0.0
+ provider.vsphere v1.2.0
```

### vSphere Provider Version
See above

### Affected Resource(s)

- `vsphere_virtual_machine`

### Terraform Configuration Files

```
### Excerpted tf.json
{
  "vsphere_virtual_machine": {
    "agent_vm": {
      "clone": {
          "template_uuid": "${data.vsphere_virtual_machine.base_template.id}"
      },
      "count": "${local.count}",
      ...omitted...
      "vapp": {
          "properties": {
            "guestinfo.coreos.config.data": "${element(data.ct_config.agents.*.rendered, count.index)}"
          }
      }
    }
  }
}
```

### Expected Behavior
It recognized the new property `vapp`.

**It does work if I build `master` myself.**

### Actual Behavior

```
❯ terraform init                                                              

Initializing provider plugins...
- Checking for available provider plugins on https://releases.hashicorp.com...
- Downloading plugin for provider "vsphere" (1.2.0)...

The following providers do not have any version constraints in configuration,
so the latest version was installed.

To prevent automatic upgrades to new major versions that may contain breaking
changes, it is recommended to add version = "..." constraints to the
corresponding provider blocks in configuration, with the constraint strings
suggested below.

* provider.rancher: version = "~> 1.2"
* provider.template: version = "~> 1.0"
* provider.vsphere: version = "~> 1.2"

Terraform has been successfully initialized!

...output omitted...
❯terraform apply     

Error: vsphere_virtual_machine.agent_vm[0]: : invalid or unknown key: vapp

```

### Steps to Reproduce

Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

**Comment by vancluever** [comment]
Hey @chandras, that feature hasn’t shipped yet and is slated for 1.3.0 (which is why it works off of master).

As a sanity check I checked the CHANGELOG to make sure there wasn’t an error there and there isn’t. 😊

**Comment by charandas** [comment]
Whoops 😬 sorry @vancluever. I apologize for my lack of diligence.
